doc/api/cluster.md

```diff
@@ -467,6 +467,12 @@ changes:
 
 * `message` {Object}
 * `sendHandle` {Handle}
+* `options` {Object} The `options` argument, if present, is an object used to
```

Comment:
This is missing from the signature above

Reply:
@mscdex thanks, i fix it.

Reply:
Hello @mscdex

Reply:
@dev-313 - you need to modify the API signature at line 466 to add the `options` argument as well - like how did it for the `worker.send` API.

Reply:
@gireeshpunathil please review now.

Reply:
looks good. @mscdex - PTAL!